app/services/bulk_upload/lettings/year2022/row_parser.rb

```diff
@@ -862,8 +863,8 @@ def validate_if_log_already_exists
       errors.add(:field_97, error_message) # startdate
       errors.add(:field_98, error_message) # startdate
       errors.add(:field_100, error_message) # propcode
-      errors.add(:field_108, error_message) # postcode_full
-      errors.add(:field_109, error_message) # postcode_full
+      errors.add(:field_108, error_message) unless bulk_upload.needstype == 2  # postcode_full
```

Comment:
I would say that seeing as earlier on in the code you were using 
`bulk_upload.needstype != 2` I would expect this to be an `if` and not `unless` either way works but I would want them to be consistent